.golangci.yml

```diff
@@ -34,6 +35,22 @@ linters-settings:
   govet:
     # report about shadowed variables
     check-shadowing: true
+    settings:
+      printf:
+        # Additionally check chainlink custom loggers
+        funcs:
+          - (github.com/smartcontractkit/chainlink-common/pkg/logger.Logger).AssumptionViolationf
+          - (github.com/smartcontractkit/chainlink-common/pkg/logger.Logger).Tracef
```

Comment:
From my other comment:
- I think we can drop these two lines (from below too)
- But I wonder if we need to repeat all of these Logger methods for SugaredLogger too, since it extends that interface

Reply:
Fortunately, there is no need to repeat. Here is a minimal example:

```
package main

import "github.com/smartcontractkit/chainlink-common/pkg/logger"

func main() {
	ll, _ := logger.New()
	ll.Infof("Type = %d", "zzz")
        ll.Infow("Type", "zzz")
	sl := logger.Sugared(ll)
	sl.Infof("Type = %d", "zzZ")
        sl.Infow("Type", "zzz") 
}
```

If you run `golangci-lint run` on this file with a single rule (for loggercheck)

- (github.com/smartcontractkit/chainlink-common/pkg/logger.Logger).Infow

or (for printf)
 
- (github.com/smartcontractkit/chainlink-common/pkg/logger.Logger).Infof

you will catch both violations.